src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java

```diff
@@ -205,6 +207,51 @@ private FilterCondition getLaunchCondition(Long launchId) {
 		return FilterCondition.builder().eq(CRITERIA_ITEM_LAUNCH_ID, String.valueOf(launchId)).build();
 	}
 
+	private void updateFilter(Filter filterable, String path) {
```

Comment:
any comment describing that 'update' does?

Reply:
unit test for this

Reply:
updated. JUnit test provided

Reply:
https://github.com/reportportal/service-api/pull/1252/files#diff-89fa7202cc0ee99329af0a05299990aa